eng/common/pipelines/templates/steps/verify-restapi-spec-location.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
+- name: ServiceDirectory
+  type: string
+  default: ''
+- name: PackageName
+  type: string
+  default: ''
+- name: ArtifactLocation
+  type: string
+  default: ''
+
+steps:
+  - task: Powershell@2
+    inputs:
+      filePath: $(Build.SourcesDirectory)/eng/common/scripts/Verify-RestApiSpecLocation.ps1
+      arguments: >
+        -ServiceDirectory "${{ parameters.ServiceDirectory }}"
+        -PackageName "${{ parameters.PackageName }}"
+        -ArtifactLocation: "${{ parameters.ArtifactLocation }}"
+        -GitHubPat "${{  parameters.GitHubSpecRepoPat }}"
```

Comment:
Why do we need to set it here as well as the env? Als it doesn't look like this parameter is listed in the template.

Reply:
Thanks, removed. Forgot to remove this line.